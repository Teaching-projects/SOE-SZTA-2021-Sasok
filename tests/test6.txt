369
36
13
10
10000
2
50
357
31
15