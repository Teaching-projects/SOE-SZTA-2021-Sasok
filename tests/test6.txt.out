Player HP-ja: Player DMG-je: Player AS-je: Player DMG növekedése: Player HP növekedése: Player AS szorzoja: Player Szintlépéshez szükséges xp mennyisége: Második egység HP-ja: Második egység DMG-je: Második egység AS-je: 
A Csata elkezdődött! Harcos és Sámán megtámadták egymást. életük: Harcos 338, Sámán 321
Harcos a(z)2. szintre lépett
Harcos megtámadta Sámán, így Sámán élete - 285
Sámán megtámadta Harcos, így Harcos élete - 10307
Harcos megtámadta Sámán, így Sámán élete - 239
Sámán megtámadta Harcos, így Harcos élete - 10276
Harcos a(z)3. szintre lépett
Harcos megtámadta Sámán, így Sámán élete - 193
Sámán megtámadta Harcos, így Harcos élete - 20245
Harcos a(z)4. szintre lépett
Harcos megtámadta Sámán, így Sámán élete - 137
Sámán megtámadta Harcos, így Harcos élete - 30214
Harcos a(z)5. szintre lépett
Harcos megtámadta Sámán, így Sámán élete - 71
Sámán megtámadta Harcos, így Harcos élete - 40183
Harcos a(z)6. szintre lépett
Harcos megtámadta Sámán, így Sámán élete - -5

Harcos GYOZEDELMESKEDETT!
